internal/store/networkpolicy_test.go

```diff
@@ -0,0 +1,59 @@
+package store
```

Comment:
Missing copyright header.

Reply:
Good catch @LiliC! I am surprised we don't have a Copyright header checker added to our linting workflows.